### DISCUSSION TITLE
Frigate+ "Report as button" missing, images super large, cannot get to the button.

### DISCUSSION THREAD

**Discussion by voron69-bit** [blakeblackshear/frigate#14155] [General Support]
When I go to Frigate.local/plus and I click on an image to report to frigate plus, all my submissions are super blown up, and I can't access the buttons to send the report. I've tried Brave, Chrome, and Edge, and all versions of Private mode. Deleted site cache and cookies, still occurs. 

![image](https://github.com/user-attachments/assets/79a50db9-9701-46ad-ac8b-c7da2854e3d3)

**Comment by voron69-bit** [comment]
I've tried scrolling out, etc. Nothing seems to work.

**Comment by hawkeye217** [comment]
Closing as a duplicate of https://github.com/blakeblackshear/frigate/discussions/13663
